align/main.py

```diff
@@ -33,6 +36,59 @@ def build_steps( flow_start, flow_stop):
     return steps_to_run
 
 
+def gen_more_primitives( primitives, topology_dir, subckt):
```

Comment:
@kkunal1408 @arvuce22  This is the code that needs upstream in the flow, either topology, or primitives or both.